packages/core/test/preview.spec.ts

```diff
@@ -42,6 +49,30 @@ const expectedCreate2: Create2ActionInput = {
   contracts: [],
   gas: '0',
 }
+const expectedCreate: CreateActionInput = {
+  actionType: ActionInputType.CREATE,
```

Comment:
It's possible to transfer value during a contract deployment, and I've added logic to the preview to display that properly. I've added this `CreateActionInput` to test that scenario is properly handled in the preview.